plugins/gatsby-plugin-auto-index-pages/gatsby-node.js

```diff
@@ -150,13 +182,31 @@ exports.createPages = async ({ actions, graphql, reporter }, pluginOptions) => {
     locales.forEach(({ localizedPath }) => {
       const localizedSlug = path.join(`/${localizedPath}`, slug);
 
+      const localeDir = dir;
+
+      localeDir.children.forEach((child) => {
+        if (child.type === 'file') {
```

Comment:
You might want to `filter` the list before looping over it. That way you don't need an `if` statement.

```js
dir.children
  .filter((child) => child.type === 'file')
  .forEach((child) => {
    // do stuff here
  });